homeassistant/components/unifiprotect/manifest.json

```diff
@@ -0,0 +1,16 @@
+{
+  "domain": "unifiprotect",
```

Comment:
Our standard to separate words with underscore in the domain name. The brand name seems to be UniFi Protect.

Reply:
That would be a breaking change for the HACS users migrating over.

Reply:
Ok. I guess we'll just have to live with it then.

Reply:
Is there a way to safely change the domain without having a user set everything back up? I am guessing there is not. Though, what would be the process for a user to migrate to the official integration? Could just they update to 2022.2 and uninstall the HACS integration at the same time?

Reply:
They may be able to uninstall while still keeping the config entry. It's not a common scenario.

We don't recommend publishing to HACS if the integration with the same domain will ever be submitted to core. It's prone for problems.

Reply:
We can add a deprecated notice to the HACS version like we did with isy994 https://github.com/shbatm/hacs-isy994#depreciated and ask users to remove it before we eventually drop it from HACS.

Unless it is has changed recently, the config entry will survive removal of the custom component via HACS.

Reply:
Yeah, that was plan to do to deprecate the HACS one once 2022.2 comes out. But I think we still need to keep the domain for the people coming from the HACS version since the config entry is tied to the domain name, right?

Reply:
Yes.  I think we are stuck with the existing domain and considering how high the usage is, I wouldn't want to change it.